includes/class-social.php

```diff
@@ -56,7 +56,7 @@ public function form( $instance ) {
 
 		foreach ( $fields as $key => $field ) {
 
-			if ( ! isset( $field['social'] ) ) {
+			if ( ! isset( $field['social'] ) || $this->is_social_media_deprecated( $field ) ) {
```

Comment:
@EvanHerman I don't think we want to do this either. If the field is currently in use, we should still show it in the widget form.